gsa/src/web/components/dialog/composercontent.js

```diff
@@ -40,6 +42,20 @@ export const COMPOSER_CONTENT_DEFAULTS = {
   includeOverrides: YES_VALUE,
 };
 
+export const ThresholdMessage = threshold => {
+  return (
```

Comment:
Could be simplified and put into an own js module. Also it isn't a component. When fixing the props params the proptypes need to be added too.
```js
const ThresholdMessage = ({threshold}) => (
  <ErrorContainer>
```